### PR TITLE
V3/vc build 0010

### PR DESCRIPTION
**Pull request by krankenbro**
### Fixed
- An issue with opencover that fails when it runs not on build server.
### Changed
- NUKE Execution Engine updated to 0.24.10
- Removed GitVersion dependency
- The Version is going to be got from Project properties now instead of GitVersion

### CODE REVIEW

File: build/_build.csproj [hunk 1]
```diff
@@ -10,19 +10,19 @@
         <ToolCommandName>vc-build</ToolCommandName>
         <NoWarn>CS0649;CS0169</NoWarn>
         <PackageOutputPath>../artifacts</PackageOutputPath>
+        <PackageIconUrl>https://github.com/VirtoCommerce/vc-community/raw/master/NuGet/icon.png</PackageIconUrl>
+        <Title>VirtoCommerce build CLI</Title>
+        <Description>Cross-platform  build automation system for VirtoCommerce projects</Description>
         <NeutralLanguage>en-US</NeutralLanguage>
         <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+        <PackageVersion>3.0.0-beta0010</PackageVersion>
```
Comment (tatarincev) [inline build/_build.csproj:18]: Please use Directory.Build.props in the build folder to increment version

File: build/_build.csproj [hunk 2]
```diff
@@ -10,19 +10,19 @@
         <ToolCommandName>vc-build</ToolCommandName>
         <NoWarn>CS0649;CS0169</NoWarn>
         <PackageOutputPath>../artifacts</PackageOutputPath>
+        <PackageIconUrl>https://github.com/VirtoCommerce/vc-community/raw/master/NuGet/icon.png</PackageIconUrl>
```
Comment (tatarincev) [inline build/_build.csproj:13]: These properties are defined in Directory.Build.props in the build folder

File: build/Build.cs [hunk 3]
```diff
@@ -150,15 +184,16 @@ void ErrorLogger(OutputType type, string text)
       .DependsOn(Test)
       .Executes(() =>
       {
-          //For platform take nuget package description from Directory.Build.Props 
+          //For platform take nuget package description from Directory.Build.Props
+          var version = BuildNumber.IsNullOrEmpty() ? PackageVersion : $"{PackageVersion}.{BuildNumber}";
```
Comment (tatarincev) [inline build/Build.cs:188]: why do not define the property ProjectVersion and reuse it in many places to follow the  DRY principle and remove code duplication?

File: build/Build.cs [hunk 4]
```diff
@@ -94,30 +99,60 @@ public static int Main()
...
+                return string.Join("-", ModuleManifest.Version, ModuleManifest.VersionTag);
+            }
+            return ModuleManifest.Version;
+        }
+    }
 
     AbsolutePath ModuleOutputDirectory => ArtifactsDirectory / (ModuleManifest.Id + ModuleSemVersion);
 
-    string ZipFileName => IsModule ? ModuleManifest.Id + "_" + string.Join("-", ModuleManifest.Version, ModuleManifest.VersionTag) + ".zip" : "VirtoCommerce.Platform." + GitVersion.SemVer + ".zip";
+    string ZipFileName => IsModule ? ModuleManifest.Id + "_" + ModuleSemVersion + ".zip" : "VirtoCommerce.Platform." + PackageVersion + (BuildNumber.IsNullOrEmpty() ? "" : $".{BuildNumber}") + ".zip";
```
Comment (tatarincev) [inline build/Build.cs:135]: contains code  duplication PackageVersion + (BuildNumber.IsNullOrEmpty() ? "" : $".{BuildNumber}") +

Comment (krankenbro) [reply build/Build.cs]: Fixed

**Pull request merged**